static/css/topic_page.scss

```diff
@@ -92,4 +92,40 @@ $chart-container-top-margin: 20px;
 .highlight-tile .desc {
   padding-left: .4em;
   font-size: 1.2rem;
+}
+
+.scatter-tooltip {
```

Comment:
we should try to align all our tooltips site wide :) are these truly specific to scatter? if so, we can move it to a scatter specific scss file.

Reply:
I agree with aligning all our tooltips site wide, but that will be for a future pr (right now we have different tooltips for map, scatter, line chart, and stat var hierarchy)

For now, I've moved to a scatter specific scss file.

Reply:
yes, definitely in a future pr